curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/replace-loops-using-recursion.english.md

```diff
@@ -7,7 +7,7 @@ videoUrl: 'https://www.freecodecamp.org/news/how-recursion-works-explained-with-
 
 ## Description
 <section id='description'>
-Recursion is the concept that a function can be expressed in terms of itself. To help understand this, start by thinking about the following task: multiply the first <code>n</code> elements in an array to create the product of the elements. Using a <code>for</code> loop, you could do this:
+Recursion is the concept that a function can be expressed in terms of itself. To help understand this, start by thinking about the following task: multiply the first elements (from 0 to <code>n</code> inclusive) in an array to create the product of the elements. Using a <code>for</code> loop, you could do this:
```

Comment:
After reading the 2nd sentence again (see below), I am not sure what the word "first" is doing there.
>To help understand this, start by thinking about the following task: multiply the **first** elements (from 0 to <code>n</code> inclusive) in an array to create the product of the elements.

Reply:
Good point, none of the firsts make sense any more.  I got rid of them both and tweaked the wording in general.